### DISCUSSION TITLE
I think I find a little BUG  for informations on moment with Templater

### DISCUSSION THREAD

**Discussion by MugenLee** [SilentVoid13/Templater#371] [Help] [answered]
First, em, sorry my English is not very good, So, please  just take a look...

The Bug is that:

when I use "tp.date.weekday" to change my "WeeklyNote", I find it can not support the "Week of Year". whether"WW","Wo","W","ww","wo",or "w"

like this imge: I want to  position in "47W", but no matter how I enter it, it is always positioned in the first week of November
![image](https://user-images.githubusercontent.com/91043731/133958011-835f0483-16dd-436a-af31-52f249066e80.png)

But when I add the "day" with moment "DD",it can position in "47W"
![image](https://user-images.githubusercontent.com/91043731/133957860-caa08112-d4a2-4da5-b718-700b372bb6df.png)

I see it than I think it can support the "Week of Year"
![image](https://user-images.githubusercontent.com/91043731/133958335-35dc0f59-64b7-4f13-bcaa-73ad32943af2.png)

So, I think it is a little bug, please take a look it, thank you !

**Comment by AB1908** [comment, marked as answer]
~~I think this is more of a locale thing.~~

See #372.

**Reply by finn-matti** [reply]
I would also guess that this has something to do with the locale. Correct me if I'm wrong, but the locale can't be overwritten at the moment.

**Reply by MugenLee** [reply]
> 我认为这更像是一个语言环境。

this is close,
pls see it
https://github.com/SilentVoid13/Templater/issues/372#issue-1001814025

**Reply by AB1908** [reply]
Can you mark this as solved? I don't have those permissions.

**Reply by MugenLee** [reply]
> 您可以将此标记为已解决吗？我没有这些权限。
yes ,but i  can not find how to do it ,can you  tell me  how to do?
